lib/network-exporters/utils/general.js

```diff
@@ -52,7 +52,12 @@ export const getFileExtension = (formatterType) => {
  * Generate a filename prefix based on the session in the format:
  * `{caseId}_{sessionId}` 
  */
-export const getFilePrefix = (session) => `${sanitizeFilename(session.sessionVariables[caseProperty])}_${session.sessionVariables[sessionProperty]}`;
+export const getFilePrefix = (session) => {
+  if (!session.sessionVariables[caseProperty]) {
+    return `${sanitizeFilename(session.sessionVariables[sessionProperty])}`;
+  }
+  return `${sanitizeFilename(session.sessionVariables[caseProperty])}_${session.sessionVariables[sessionProperty]}`;
+}
```

Comment:
Now that `caseProperty` can potentially be null, this is perhaps best implemented as nullish coalescence:

```
const caseIdentifier = session.sessionVariables[caseProperty] ?? session.sessionVariables[sessionProperty];

${sanitizeFilename(caseIdentifier)}
```

Up to you if you think this is more readable, but personally I think it is.